skimage/feature/_canny.py

```diff
@@ -188,6 +192,9 @@ def canny(image, sigma=1., low_threshold=None, high_threshold=None,
     # mask by one and then mask the output. We also mask out the border points
     # because who knows what lies beyond the edge of the image?
 
+    if np.issubdtype(image.dtype, np.int64) or np.issubdtype(image.dtype, np.uint64):
+        raise ValueError("skimage does not support 64-bit integer images")
```

Comment:
```suggestion
    if np.issubdtype(image.dtype, np.int64) or np.issubdtype(image.dtype, np.uint64):
        raise ValueError("64-bit integer images are not supported")
```
I wasn't even aware that we didn't support int64 officially. Nevertheless, I would think that this suggestion might age better in case we get rid of the scaling. In that case https://github.com/scikit-image/scikit-image/issues/4442#issuecomment-584897639 suggest that we could support int64 just fine as long as the function at hand permits it?

Reply:
Yes, there's nothing in principle that stops us from supporting int64 other than that scaling. But, unfortunately the scaling is baked into canny and it's non-trivial to remove. I'm not even sure it's possible to remove the scaling without changing results, unless you look at the image values themselves. A nice task for v2, perhaps, if you want to file the issue already.

Reply:
I agree with you. Even if scikit-image supported int64 at some future point, this still wouldn't work for canny. Though, this suggestion would then still be up to date; the original referencing "skimage does not support" wouldn't.

Reply:
Oh, right, you wanted me to update the text; sorry I missed that!